Azure-Samples/chat-with-your-data-solution-accelerator
# fix: Removing index creation from init of AzureSearchHelper.py

## Purpose

* This PR fixes a bug: First message in web chat creates an index for Push approach although IV is enabled while creating the conversation logger index.(Index creation called in init) 

## Does this introduce a breaking change?



## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

* When IV is enabled, web chat message should not create the index.
